test/contracts/IncentivePool/update.test.ts

```diff
@@ -0,0 +1,250 @@
+import hre from 'hardhat';
+
+import { ethers, waffle } from 'hardhat';
+import ElyfiContracts from '../../types/ElyfiContracts';
+import { BigNumber, utils } from 'ethers';
+import { setupAllContracts } from '../../utils/setup';
+import { RAY } from '../../utils/constants';
+import { getIncentivePoolData, getUserIncentiveData } from '../../utils/Helpers';
+import { expectIncentiveDataAfterDeposit } from '../../utils/Expect';
+import { advanceTimeTo, getTimestamp } from '../../utils/time';
+import IncentivePoolData from '../../types/IncentivePoolData';
+import UserIncentiveData from '../../types/UserIncentiveData';
+import { expect } from 'chai';
+import { IncentivePool, IncentivePool__factory } from '../../../typechain';
+import { testIncentiveAmountPerSecond } from '../../utils/testData';
+import { calculateIncentiveIndex, calculateUserIncentive } from '../../utils/calculations';
+require('../../assertions/equals.ts');
+
+describe('', () => {
+  let elyfiContracts: ElyfiContracts;
+  let newIncentivePool: IncentivePool;
+
+  const provider = waffle.provider;
+  const [deployer, account0, account1] = provider.getWallets();
+
+  const amount = ethers.utils.parseEther('1');
+
+  beforeEach('', async () => {
+    elyfiContracts = await setupAllContracts();
+
+    const incentivePoolFactory = (await ethers.getContractFactory(
+      'IncentivePool'
+    )) as IncentivePool__factory;
+
+    newIncentivePool = await incentivePoolFactory.deploy(
+      deployer.address,
```

Comment:
@donguks 
When deploying new incentivePool, the moneyPool address in constructor argument should be set to the EOA account, like deployer